packages/web-components/src/components/va-memorable-date/test/va-memorable-date.e2e.ts

```diff
@@ -1262,64 +1281,27 @@ describe('va-memorable-date', () => {
     });
   });
 
-  // Begin USWDS v3 test with monthSelect prop
-  it('uswds v3 renders with monthSelect', async () => {
+  // Begin USWDS v3 test without monthSelect prop
+  it('uswds v3 renders without monthSelect', async () => {
```

Comment:
The default is now to have a `va-memorable-date` with `monthSelect` enabled, and it's now up to the user to set `monthSelect=false` if they want a text input. So that means all examples by default have the `monthSelect` input, so I modified this example to show one with the month text input